<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日期时间设置</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="FDBGroupBox">
     <property name="title">
      <string>来自因子库</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_18">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>起始时点</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>StartDTEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QDateTimeEdit" name="StartDTEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_21">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>终止时点</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>EndDTEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QDateTimeEdit" name="EndDTEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>因子表</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>FTComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>因子</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>FactorComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="FactorComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_26">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>IDEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="IDEdit">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="FDTSelectTypeComboBox">
        <item>
         <property name="text">
          <string>覆盖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>并集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>交集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>左差右</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>右差左</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>对称差</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="SelectFDBDTButton">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QComboBox" name="FTComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QListWidget" name="DateTimeListWidget">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="DateSetupGroupBox">
     <property name="title">
      <string>日期设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QDateEdit" name="StartDateEdit">
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="DatePeriodComboBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="7">
       <widget class="QListWidget" name="DateListWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;终止日期&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>EndDateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="SelectDateButton">
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDateEdit" name="EndDateEdit">
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;起始日期&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>StartDateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;日期周期&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>DatePeriodComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;日期类型&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>DateTypeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;交易所&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>ExchangeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="DateTypeComboBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="ExchangeEdit">
        <property name="text">
         <string>SSE</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="DateSelectTypeComboBox">
        <item>
         <property name="text">
          <string>覆盖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>并集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>交集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>左差右</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>右差左</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>对称差</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="TimeSetupGroupBox">
     <property name="title">
      <string>时间设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;AM 起始时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>AMStartTimeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="AMStartTimeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>9</hour>
          <minute>30</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumTime">
         <time>
          <hour>11</hour>
          <minute>59</minute>
          <second>59</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="6">
       <widget class="QListWidget" name="TimeListWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;AM 终止时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>AMEndTimeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="AMEndTimeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>11</hour>
          <minute>30</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumTime">
         <time>
          <hour>11</hour>
          <minute>59</minute>
          <second>59</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;PM 起始时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>PMStartTimeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="PMStartTimeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>13</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumTime">
         <time>
          <hour>12</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;PM 终止时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>PMEndTimeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTimeEdit" name="PMEndTimeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>15</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumTime">
         <time>
          <hour>12</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;时间周期&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>TimePeriodEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTimeEdit" name="TimePeriodEdit">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>1</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumTime">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>1</second>
         </time>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="SelectTimeButton">
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="TimeSelectTypeComboBox">
        <item>
         <property name="text">
          <string>覆盖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>并集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>交集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>左差右</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>右差左</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>对称差</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="DTSelectTypeComboBox">
       <item>
        <property name="text">
         <string>覆盖</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>并集</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>交集</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>左差右</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>右差左</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>对称差</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SelectDTButton">
       <property name="maximumSize">
        <size>
         <width>88</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>时点数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>DateTimeNumEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DateTimeNumEdit">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AcceptButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RejectButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>AcceptButton</zorder>
  <zorder>RejectButton</zorder>
  <zorder>DateSetupGroupBox</zorder>
  <zorder>SelectDTButton</zorder>
  <zorder>DateTimeListWidget</zorder>
  <zorder>label_25</zorder>
  <zorder>DateTimeNumEdit</zorder>
  <zorder>TimeSetupGroupBox</zorder>
  <zorder>DTSelectTypeComboBox</zorder>
  <zorder>FDBGroupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
